### ISSUE TITLE
Remove some vulnerabilities reported which are not useful

### ISSUE THREAD

**Issue by remidinishanth** [aquasecurity/trivy#442]
When we scan centos:7 image with trivy( 0.5.2), it reports vulnerabilities 

```
centos:7 (centos 7.7.1908)
==========================
Total: 642 (UNKNOWN: 0, LOW: 65, MEDIUM: 468, HIGH: 106, CRITICAL: 3)
```

out of which the following CVEs are reported as won’t fix by RedHat

https://access.redhat.com/security/cve/cve-2014-9939
https://access.redhat.com/security/cve/cve-2019-9924
https://access.redhat.com/security/cve/cve-2017-13716
https://access.redhat.com/security/cve/cve-2017-14930
https://access.redhat.com/security/cve/cve-2017-7614

I think there are many such CVEs reported like these, can we do something about these?

**Comment by knqyf263** [comment]
You can use `--ignore-unfixed"
https://github.com/aquasecurity/trivy#ignore-unfixed-vulnerabilities

**Comment by remidinishanth** [comment]
But do we miss any critical security vulnerabilities by using this option? If they are critical, they would be fixed, so this is a workaround?

**Comment by knqyf263** [comment]
@remidinishanth Yes, if the vulnerability is critical and not fixed, `--ignore-severity` makes you miss such vulnerability, but the critical vulnerability is usually supposed to be fixed. Or, you can combine two outputs. `trivy --severity CRITICAL -f json -o result1.json` and `trivy --ignore-unfixed -f json -o result2.json`.
This is not perfect, but you can use it as a workaround.

**Comment by dinvlad** [comment]
Is it possible to still report all vulnerabilities, but indicate in the report if a given vuln is unfixed? I checked JSON output, but it doesn't seem to include this information, unfortunately…

**Comment by aristosvo** [comment]
I think it has: `trivy -q -f json docker.io/bitnami/mongodb:4.4.8-debian-10-r0  | jq '.[].Vulnerabilities[].FixedVersion' | grep -v null`

**Comment by dinvlad** [comment]
@aristosvo thanks, you're right! I think I was only looking at the unfixed ones, and `FixedVersion` does not appear in those, not even with a `null` value. I think it could be helpful for parsers to include `"FixedVersion": null` for consistency.

**Comment by knqyf263** [comment]
We already have state in our database. We'll export it to the result. Thanks for the feedback.
